src/components/modals/SubmitTranscriptModal.tsx

```diff
@@ -35,6 +36,7 @@ type Props = {
 const SubmitTranscriptModal = ({ submitState, onClose }: Props) => {
   const { stepIdx, steps, isLoading, isError, isModalOpen, prResult, err } =
     submitState;
+  const errorMessage = err?.response?.data?.message || err?.message;
```

Comment:
It's easier to manage error as `Error`, if axios returns an Error then in the function that axios is called throw a new Error and pass the axios error message .